.env.example

```diff
@@ -1,6 +1,6 @@
-APP_NAME=Laravel
+APP_NAME=DiscordClone
 APP_ENV=local
-APP_KEY=
+APP_KEY=base64:/6RMPYzR0XkBwFxkp8592fYoHey8Y+t9uUnucYF/pls=
```

Comment:
Silly thing but never generate a base key for an example environment. Probably you forgot to remove it.